JetNews/build.gradle

```diff
@@ -31,6 +31,6 @@ allprojects {
     repositories {
         google()
         jcenter()
-        maven { url 'https://ci.android.com/builds/submitted/6202563/androidx_snapshot/latest/ui/repository/' }
+        maven { url 'https://ci.android.com/builds/submitted/6204302/androidx_snapshot/latest/ui/repository/' }
```

Comment:
If we're using `dev05` we should remove these :)

Reply:
And in the app build gradle file, let's change the compilerOptions to use `dev05` as well

Reply:
This uses the same snapshot as dev05, but does not yet target dev05.

However, I'm going to change that in the next commit and point directly to dev05 since snapshots are broken this morning.

Will remove this line, can re-add on develop to switch back to a snapshot.